### PR TITLE
Replace aggregatable-specific source headers with optional fields on Register-Source header

### PR DESCRIPTION
**Pull request by apasel422**
#432

### CODE REVIEW

**Review by linnan-github — APPROVED** [review]

**Comment by linnan-github** [comment]
For the naming, probably aggregation_keys? For contributions, I would expect values to be included.

**Comment by apasel422** [comment]
> For the naming, probably aggregation_keys? For contributions, I would expect values to be included.

That's fine. I was just basing `contributions` on the wording from https://github.com/WICG/conversion-measurement-api/blob/main/AGGREGATE.md#attribution-source-registration:

> This defines a list named histogram contributions

File: AGGREGATE.md [hunk 1]
```diff
@@ -558,7 +563,7 @@ The server can respond with an optional header
 value.
 
 ```http
-Attribution-Reporting-Register-Aggregatable-Source: [{....}]
+Attribution-Reporting-Register-Source: [{..., "aggregation_keys": ...}]
 Attribution-Reporting-Alternative-Aggregation-Mode: "experimental-poplar"
```
[the remark this answers is not in the document]
Comment (apasel422) [reply AGGREGATE.md:567]: I don't know anything about this header. It doesn't seem to have been implemented in Chrome. Should it be removed?

**Review by linnan-github — CHANGES_REQUESTED** [review]

File: AGGREGATE.md [hunk 2]
```diff
@@ -69,25 +69,30 @@ example an ad-tech will use the API to collect:
 * Aggregate purchase values at a per geo level 
 ### Attribution source registration
 
-Registering sources eligible for aggregate reporting entails adding a new header
-on the response to the `attributionsrc` request:
-`Attribution-Reporting-Register-Aggregatable-Source`, in the form of a JSON
-list.
+Registering sources eligible for aggregate reporting entails adding a new
+`aggregatable_source` list field to the JSON dictionary of the
```
[the remark this answers is not in the document]
Comment (apasel422) [reply AGGREGATE.md]: I should've read your suggestion more closely. It removed a bunch of words unintentionally.

**Review by linnan-github — APPROVED** [review]

**Review by csharrison — APPROVED** [review]

File: AGGREGATE.md [hunk 3]
```diff
@@ -69,25 +69,30 @@ example an ad-tech will use the API to collect:
...
-},
 {
-  // Generates a "0x5" key piece (low order bits of the key) for the key named "geoValue"
-  "id": "geoValue",
-  // Source-side geo region = 5 (US), out of a possible ~100 regions.
-  "key_piece": "0x5"
-}]
+  ... // existing fields, such as `source_event_id` and `destination`
+
+  "aggregation_keys": [
```
Comment (johnivdel) [inline AGGREGATE.md:79]: Should this be aggregatable_keys to match the rest of the naming? Ditto for the prose

**Review by johnivdel — COMMENTED** [review]
lgtm with comment

**Review by johnivdel — APPROVED** [review]

**Pull request merged**